backend/eAkademik.API/Services/UserService.cs

```diff
@@ -52,4 +53,28 @@ public async Task EditUser(UserViewModel user)
 
         await _context.SaveChangesAsync();
     }
+
+    public async Task<User> CreateUser(string firstName, string lastName, string email, string password)
+    {
+        if (!new EmailAddressAttribute().IsValid(email))
```

Comment:
Fajna logika podoba mi się. Gdy stworzysz ViewModel dop tworzenia użytkownika stworzyłbym tam metodę która sprawdza wewnętrzy stan klas sprawdzającą poprawność emailu. Później można by dodać metodę np `isValid` która wywoła nie tylko sprawdzenie poprawności adresu email ale np. czy hasło nie jest puste itp

Reply:
@JacobChwastek Czy takie isValid jest ok? 
![image](https://user-images.githubusercontent.com/60351575/168418047-d1434b35-682e-4ca9-a057-dea3f7b7265f.png)

Reply:
Myślałem też nad użyciem ValidationResult, ale chyba to nie jest do tego? https://docs.microsoft.com/pl-pl/dotnet/api/system.windows.controls.validationresult?view=windowsdesktop-6.0